pkg/python/packaging/parse.go

```diff
@@ -26,11 +27,26 @@ func (*Parser) Parse(r dio.ReadSeekerAt) ([]types.Library, []types.Dependency, e
 		return nil, nil, xerrors.Errorf("read MIME error: %w", err)
 	}
 
+	license := h.Get("License-Expression")
+	if license == "" {
+		license = h.Get("License")
+	}
+	if license == "" {
+		for _, classifier := range h.Values("Classifier") {
+			if strings.HasPrefix(classifier, "License :: ") {
+				values := strings.Split(classifier, " :: ")
+				license = values[len(values)-1]
+				break
+			}
+		}
+	}
+
 	return []types.Library{
 		{
-			Name:    h.Get("Name"),
-			Version: h.Get("Version"),
-			License: h.Get("License"),
+			Name:        h.Get("Name"),
+			Version:     h.Get("Version"),
+			License:     license,
+			LicenseFile: h.Get("License-File"),
```

Comment:
It could be a bit hacky, but I'm not keen on adding a new field only for python. What if adding `fil://license.txt` to `License` if the license is empty?

Reply:
let's try this way.